I think this line will need changing:

https://github.com/pandas-dev/pandas/blob/39e0964ad7896ed758fb1f8fb7c2326685277b5e/.devcontainer.json#L13

I tried this out on VSCode, but couldn't build the C extensions:
```console
root@6d1d5c4a30b0:/workspaces/pandas-dev# git clean -fxfd pandas
root@6d1d5c4a30b0:/workspaces/pandas-dev# python setup.py build_ext -j 1
root@6d1d5c4a30b0:/workspaces/pandas-dev# python -c 'import pandas'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/workspaces/pandas-dev/pandas/__init__.py", line 22, in <module>
    from pandas.compat import is_numpy_dev as _is_numpy_dev  # pyright: ignore # noqa:F401
  File "/workspaces/pandas-dev/pandas/compat/__init__.py", line 25, in <module>
    from pandas.compat.numpy import (
  File "/workspaces/pandas-dev/pandas/compat/numpy/__init__.py", line 4, in <module>
    from pandas.util.version import Version
  File "/workspaces/pandas-dev/pandas/util/__init__.py", line 2, in <module>
    from pandas.util._decorators import (  # noqa:F401
  File "/workspaces/pandas-dev/pandas/util/_decorators.py", line 14, in <module>
    from pandas._libs.properties import cache_readonly
  File "/workspaces/pandas-dev/pandas/_libs/__init__.py", line 13, in <module>
    from pandas._libs.interval import Interval
ModuleNotFoundError: No module named 'pandas._libs.interval'
```

--- a/.devcontainer.json
+++ b/.devcontainer.json
@@ -9,8 +9,7 @@
 	// You can edit these settings after create using File > Preferences > Settings > Remote.
 	"settings": {
 		"terminal.integrated.shell.linux": "/bin/bash",
-		"python.condaPath": "/opt/conda/bin/conda",
-		"python.pythonPath": "/opt/conda/bin/python",
+		"python.pythonPath": "/usr/local/bin/python",
 		"python.formatting.provider": "black",
 		"python.linting.enabled": true,
 		"python.linting.flake8Enabled": true,

